apps/api/server.js

```diff
@@ -0,0 +1,86 @@
+const express = require('express')
+const app = express()
+const port = 3001
+var cors = require('cors')
+var bodyParser = require('body-parser')
+
+app.use(cors()) // Use this after the variable declaration
+app.use(bodyParser.json());
+
+const corsOptions ={
+    origin:'http://localhost:3000', 
+    credentials:true,            //access-control-allow-credentials:true
+    optionSuccessStatus:200
+}
+app.use(cors(corsOptions));
+
+const exercises = [
```

Comment:
You can try to read exercices from the database know rather than having an hard coded object